github/Organization.py

```diff
@@ -1292,15 +1122,18 @@ def create_migration(
         """
         assert isinstance(repos, (list, tuple)), repos
         assert all(isinstance(repo, str) for repo in repos), repos
-        assert lock_repositories is github.GithubObject.NotSet or isinstance(lock_repositories, bool), lock_repositories
-        assert exclude_attachments is github.GithubObject.NotSet or isinstance(
-            exclude_attachments, bool
-        ), exclude_attachments
+        assert is_optional(lock_repositories, bool), lock_repositories
+        assert is_optional(exclude_attachments, bool), exclude_attachments
         post_parameters = {"repositories": repos}
```

Comment:
Why not merging the `update` from below into this?
```suggestion
        post_parameters = NotSet.remove_unset_items(
                {
                    "repositories": repos,
                    "lock_repositories": lock_repositories,
                    "exclude_attachments": exclude_attachments,
                }
            )
```